internal/fcs/cloud_aws_account.go

```diff
@@ -303,18 +309,86 @@ func (r *cloudAWSAccountResource) Schema(
 						Default:     booldefault.StaticBool(true),
 						Description: "Set to true if a CloudTrail already exists",
 					},
+					"log_ingestion_method": schema.StringAttribute{
+						Optional:    true,
+						Computed:    true,
+						Default:     stringdefault.StaticString("eventbridge"),
+						Description: "Log ingestion method for real-time visibility. Valid values are 'eventbridge' or 's3'",
+						Validators: []validator.String{
+							stringvalidator.OneOf("eventbridge", "s3"),
+						},
+					},
+					"log_ingestion_s3_bucket_name": schema.StringAttribute{
+						Optional:    true,
+						Description: "S3 bucket name for CloudTrail log ingestion when log_ingestion_method is 's3'. Required when using S3 method",
+						Validators: []validator.String{
+							stringvalidator.Any(
+								stringvalidator.LengthBetween(0, 0),
+								stringvalidator.All(
+									stringvalidator.RegexMatches(
+										regexp.MustCompile(`^[a-z0-9][a-z0-9.-]*[a-z0-9]$`),
+										"must be 3-63 characters, contain only lowercase letters, numbers, dots, and hyphens, and start/end with alphanumeric characters",
+									),
+									stringvalidator.LengthAtLeast(3),
+									stringvalidator.LengthAtMost(63),
+								)),
+						},
+					},
```

Comment:
LMK if I am missing something, but this value should either be null or atleast 3-63 characters so there is no need too have a OR check for 0 characters.
```suggestion
					"log_ingestion_s3_bucket_name": schema.StringAttribute{
						Optional:    true,
						Description: "S3 bucket name for CloudTrail log ingestion when log_ingestion_method is 's3'. Required when using S3 method",
						Validators: []validator.String{
							stringvalidator.All(
								stringvalidator.RegexMatches(
									regexp.MustCompile(`^[a-z0-9][a-z0-9.-]*[a-z0-9]$`),
									"must be 3-63 characters, contain only lowercase letters, numbers, dots, and hyphens, and start/end with alphanumeric characters",
								),
								stringvalidator.LengthAtLeast(3),
								stringvalidator.LengthAtMost(63),
							),
						},
					},
```